### PR TITLE
Add OpenStack section, add security groups doc

### PR DESCRIPTION
**Pull request by prollynomial**

### CODE REVIEW

**Comment by prollynomial** [comment]
Should this still be merged into master? Or is dev the place to do that? I'm confused about what those 2 branches are for.

File: source/includes/_openstack.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# OpenStack Plugin
+
+The CloudMC OpenStack plugin incorporates Nova and Neutron features into CloudMC's API. It also enforces CloudMC's environment-centric multi-tenancy model and RBAC over top of OpenStack.
```
Comment (pdube) [inline source/includes/_openstack.md]: We will not be limited to Nova and Neutron. Maybe "The CloudMC OpenStack plugin incorporates various OpenStack services such as Nova and Neutron into..."

File: source/includes/openstack/_securityGroups.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
+### Security groups
+
+Security groups manage network access to instances.
+
+#### List security groups
+
+```shell
+curl -H "MC-Api-Key: your_api_key" \
+    "https://api.your.cloudmc/v1/services/compute-os/devel/securityGroups"
```
Comment (pdube) [inline source/includes/openstack/_securityGroups.md]: securitygroups

File: source/includes/_openstack.md (hunk 1 above)
Comment (prollynomial) [inline source/includes/_openstack.md]: Done

File: source/includes/openstack/_securityGroups.md (hunk 2 above)
Comment (prollynomial) [inline source/includes/openstack/_securityGroups.md]: Done

**Pull request merged**